### ISSUE TITLE
Error when creating byte string

### ISSUE THREAD

**Issue by jeffThompson** [sole/snippets#2]
In line 67 of the [`Generate Noise` Python script](https://github.com/sole/snippets/blob/master/audio/generate_noise_test_python/script.py), I get an error.

If I change it to `value_str = b''.join(values)` it works great – don't know if this is because I'm using Python 3 but might want to consider changing it.

Thanks for the great tutorial!

**Comment by carbonsoda** [comment]
Yeah, this is an [issue with Python 3](https://docs.python.org/3/howto/pyporting.html#text-versus-binary-data). 

Since it's just a snippet, it's probably best to just insert what you wrote and have the Python 2 line commented out -- so 2.7 folks can still enjoy the tutorial too😄
